app/src/main/res/layout/content_main.xml

```diff
@@ -41,9 +45,10 @@
         app:layout_constraintVertical_bias="0.113" />
 
     <TextView
+        android:id="@+id/textView2"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:text="Welcome!"
+        android:text="@string/welcome"

```

Comment:
Created string in strings.xml and replaced text with @string. This removed this warning in android studio about using string values.

Reply:
ditto above.  Great idea.